.github/CODEOWNERS

```
@@ -177,6 +177,7 @@ packages/react-components/react-avatar-context @microsoft/teams-prg
 packages/react-components/react-infobutton @microsoft/cxe-red @sopranopillow
 packages/react-migration-v8-v9 @microsoft/cxe-coastal @geoffcoxmsft
 packages/react-components/react-tree @microsoft/teams-prg
+packages/react-components/react-skeleton @microsoft/cxe-red
```

You should add yourself since you are the owner of the component :)

```suggestion
packages/react-components/react-skeleton @microsoft/cxe-red @tomi-msft
```